core/trino-main/src/main/java/io/trino/exchange/ExchangeManagerRegistry.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.exchange;
+
+import io.airlift.log.Logger;
+import io.trino.metadata.HandleResolver;
+import io.trino.spi.TrinoException;
+import io.trino.spi.classloader.ThreadContextClassLoader;
+import io.trino.spi.exchange.ExchangeManager;
+import io.trino.spi.exchange.ExchangeManagerFactory;
+
+import javax.annotation.concurrent.GuardedBy;
+import javax.inject.Inject;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkState;
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static io.airlift.configuration.ConfigurationLoader.loadPropertiesFrom;
+import static io.trino.spi.StandardErrorCode.EXCHANGE_MANAGER_NOT_CONFIGURED;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+public class ExchangeManagerRegistry
+{
+    private static final Logger log = Logger.get(ExchangeManagerRegistry.class);
```

Comment:
LOG

Reply:
Currently it is inconsistent across the code base. But there are 260 matches when I search for `Logger log` and only 33 when I search for `Logger LOG`, so it looks like `Logger log` is preferred.

Reply:
Yes, I'm aware of the inconsistency. But it should be `LOG` per convention since it's a static final variable. We should not contribute to the unconventional usage.

Reply:
Yeah, code style convention says `The names of variables declared class constants and of ANSI constants should be all uppercase with words separated by underscores ("_").`: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

However whether a `Logger` can be considered a constant is controversial. Generally constants are expected to be inherently immutable objects. Declaring something as `private static final` does not make something to be a constant. The declaration may also be used for static fields assigned once and accessible only from within a class.

For example codestyle document from Google says:

> Constant names use CONSTANT_CASE: all uppercase letters, with each word separated from the next by a single underscore. But what is a constant, exactly?
> Constants are static final fields whose contents are deeply immutable and whose methods have no detectable side effects. This includes primitives, Strings, immutable types, and immutable collections of immutable types. If any of the instance's observable state can change, it is not a constant. Merely intending to never mutate the object is not enough. 

And they explicitly mention examples that shouldn't be considered constants and declaring a logger is one of them:

> static final Logger logger = Logger.getLogger(MyClass.getName());

https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

I don't have a particularly strong opinion here. It feels either should do as long as it is used consistently. In the current state it feels like `Logger log` will be more consistent with the other places in the code base. 

If you feel particularly strong I can do the rename. Regardless we should probably follow up with a PR that would rename all other places to make it consistent and add a style check rule.